CONTRIBUTING.md

```diff
@@ -30,23 +37,25 @@ Each pull request with feature change should have accompanying documentations un
 
 ## Explicit over implicit
 
-Each package, feature, code and decision should be explicit and well documented over implicitly guessing.
+- Each package, feature, code and decision should be explicit and well documented over implicitly guessing.
+- Each test must be written explicitly as clear as possible with no implicit guessing.
 
 ## TypeScript
 
 TypeScript must be used for all code written in this project.
 
-> Heavily adapted from [deno style guide](https://github.com/denoland/deno/blob/main/docs/contributing/style_guide.md).
-
 ### Document and maintain browser compatibility.
 
 ### Minimize dependencies (target zero)
 
 ### Do not depend on external code. (never if possible)
 
-### Use underscores, not dashes in filenames.
+### Use PascalCase and period, not underscores, or dashes in filenames.
+
+Example: Use `FooBar.ts` instead of `foo-bar.ts` or `foo_bar.ts`.
```

Comment:
```suggestion
Example: Use `fooBar.ts` instead of `foo-bar.ts` or `foo_bar.ts`.
```

Maybe we should encourage, first letter capital specifically for `export default class Foobar {}`

Reply:
quite confused why what you mean since `PascalCase` encourage first letter capitalization